spec.emu

```diff
@@ -1036,7 +1031,7 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
       </emu-alg>
       <emu-grammar>ImportDeclaration : `import` ImportClause FromClause `;`</emu-grammar>
       <emu-alg>
-        1. Let _module_ be <del>the sole element of ModuleRequests</del><ins>SV</ins> of |FromClause|.
+        1. Let _module_ be the sole element of ModuleRequests of |FromClause|.
```

Comment:
I think you forgot to revert this while reverting the changes to ImportEntries that I initially suggested copying from the spec I wrote for defer eval (I like overloading `[[ImportName]]` to also be `~source~`/`~defer~`, as you did, more :) ).